targets/java/source_shared/src/test/java/com/playfab/test/PlayFabApiTest.java.ejs

```diff
@@ -455,10 +460,8 @@ public class PlayFabApiTest
         errRequest.FunctionName = "throwError";
         errRequest.PlayFabId = playFabId;
         PlayFabResult<PlayFabServerModels.ExecuteCloudScriptResult> errResult = PlayFabServerAPI.ExecuteCloudScript(errRequest);
-        VerifyResult(errResult, true);
-        assertTrue(errResult.Result.FunctionResult == null);
-        assertNotNull(errResult.Result.Error);
-        assertEquals(errResult.Result.Error.Error, "JavascriptException");
+        String errorMessage = CompileErrorsFromResult(errResult);
```

Comment:
Does errResult.GenerateErrorReport not exist?

Reply:
Nope, it looks like similar logic is in the CompileErrorsFromResult function (takes the error message and appends all error details to a single string).